requirements/tests.txt

```diff
@@ -9,5 +9,5 @@ python-dotenv
 python-gnupg>=0.5.0
 pytz
 testfixtures
-tox
+tox>=4.0.0
```

Comment:
We should create an issue to replace `tox` with `nox`.

Reply:
- #487